01week/rockPaperScissors.js

```diff
@@ -8,9 +39,30 @@ const rl = readline.createInterface({
 });
 
 
-function rockPaperScissors(hand1, hand2) {
+function rockPaperScissors(PlayerHandOnehand1, hand2) {
+
+  // Write code here  
+  //These two variables declare the two players.
+  const Hand1 = 0;
```

Comment:
These 2 variables aren't needed.  Since they are being passed into your RPS conditions and never set to any other value than 0, the function will always return a tie since 0 === 0.  You should consider using the arguments that are passed into the RPS function (PlayerHandOnehand1, hand2): these are the inputs from the user.